frontend/src/components/IssueTaskStatementPanel.vue

```diff
@@ -11,6 +11,7 @@
       >
         {{ rollback ? "Rollback SQL" : "SQL" }}
         <span v-if="create && !rollback" class="text-red-600">*</span>
+        <span class="text-accent">{{ hintForSQL }}</span>
```

Comment:
let's include the below "This is a baseline migration and bytebase won't ... " in the hint SQL as well.

By doing this, we can avoid passing migrationType as well